_docs/reference/labels.md

```diff
@@ -44,10 +44,20 @@ To add overcommit labels to a pool:
 ```bash
 storageos pool update --label-add storageos.com/overcommit=20 default
 ```
-This would result in being able to create volumes whose total capacity exceeds
-the capacity of the underlying node by 20%. It is not possible to create a 
-volume whose capacity exceeds the capacity of underlying nodes as StorageOS
-volumes must always be able to fit on an underlying node. 
+This would result in the total pool volume being increased by 20% and the
+maximum possible volume size being increased by 20%. If you have 120GB pool
+spread over 3 nodes with 40GB disks then the maximum volume size is 40GB. If
```

Comment:
I wouldn't say you have a 120GB pool spread over 3 nodes. Instead, I'd say

Even in the case of having 3 nodes with 40GB each forming a node pool of 120GB, the maximum volume size is still 40GB. Because of performance predictability and high consistency, StorageOS volumes must fit into a node entirely. 


(Because storageos pools are a storageos concepts, while nodes are well known concepts, hence simpler to understand)

Reply:
I believe I've resolved this.